contracts/campaignAirdrops/shared/CampaignAirdropsShared.sol

```diff
@@ -55,10 +57,17 @@ abstract contract CampaignAidropsShared is AirdropMerkleProof {
     function claimHandler(bytes32[] calldata _merkleProof) internal {
         if (hasClaimed[msg.sender]) revert AlreadyRedeemed();
         if (checkProof(_merkleProof, merkleRoot) == false) revert NotEligible();
+        if (msg.value != IAirBroFactory(airbroCampaignFactoryAddress).claimFee()) revert InvalidFeeAmount();
```

Comment:
You are casting airbroCampaignFactoryAddress multiple times to IAirBroFactory in this contract. Minor improvement would be to declare (on line 9)
    IAirBroFactory public immutable airbroCampaignFactory;
    and then in constructor (line 25)
    airbroCampaignFactory = IAirBroFactory(_campaignFactoryAddress);

and then you don't need to cast it in each function, you can call stuff directly